specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -32,11 +32,12 @@
   "paths": {
     "/prebuilt/receipt/asyncBatchAnalyze": {
       "post": {
-        "description": "Batch Read Receipt operation. The response contains a field called 'Operation-Location', which contains the URL that you must use for your 'Get Read Receipt Result' operation.",
-        "operationId": "BatchReadReceipt",
+        "description": "Extract field text and semantic values from a given receipt document.",
+        "operationId": "AnalyzeReceipt",
```

Comment:
Was this rename intentional? It will be a breaking change for autorest generated client libraries...

Reply:
Yes it is intentional.  We have not released the client libraries because the name was inconsistent before.